src/v0/destinations/active_campaign/transform.js

```diff
@@ -60,6 +61,7 @@ const syncContact = async (contactPayload, category, destination) => {
   const res = await httpPOST(endpoint, requestData, requestOptions, {
     destType: 'active_campaign',
     feature: 'transformation',
+    endPoint,
```

Comment:
```suggestion
    endpointPath: endPoint,
```
We are looking for [statTags.endpoinPath](https://github.com/rudderlabs/rudder-transformer/blob/66a53907539ce96d709d2d3a5e493c4ee6507402/src/adapters/network.js#L51).

Reply:
Thank you for pointing it